cli/defenseclaw/tui/panels/setup.py

```
@@ -1935,6 +1952,477 @@ def _custom_providers_wizard_fields() -> tuple[WizardFormField, ...]:
+    # operator explicitly toggles it off; --yes prevents a hidden stdin prompt
+    # on native Windows once the command has been approved.
+    args.append("--yes")
+    if wizard_bool_value(fields, "Dry Run", "yes") == "yes":
+        args.append("--dry-run")
+    if wizard_bool_value(fields, "JSON Output", "no") == "yes":
+        args.append("--json")
+    if wizard_bool_value(fields, "Restart Gateway", "no") == "yes":
+        args.append("--restart")
+
+
+def _build_redaction_args(fields: Sequence[WizardFormField]) -> tuple[str, ...]:
```

Ruff (0.16.1)

[warning] Too many branches (41 > 12)

(PLR0912)